packages/components/src/KaizenProvider/KaizenProvider.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { render } from "@testing-library/react"
+import { KaizenProvider, KaizenProviderProps } from "./KaizenProvider"
+
+const KaizenProviderWrapper = (props?: KaizenProviderProps): JSX.Element => (
```

Comment:
Usually for testing wrappers with defaults, I'd do something like this:
```suggestion
const KaizenProviderWrapper = (props?: Partial<KaizenProviderProps>): JSX.Element => (
```

But given you don't have a default value for `children` and it's required, it makes more sense to just not make it optional.
```suggestion
const KaizenProviderWrapper = (props: KaizenProviderProps): JSX.Element => (
```